executor/common_linux.h

```diff
@@ -1466,6 +1466,98 @@ static long syz_extract_tcp_res(volatile long a0, volatile long a1, volatile lon
 #include "common_usb_linux.h"
 #endif
 
+#if SYZ_EXECUTOR || __NR_syz_raw_bind_info
+#include <linux/raw.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/sysmacros.h>
+static long syz_raw_bind_info(int fd, int arg)
+{
+	// syz_raw_bind_info$char_raw(dev fd, raw_dev int32[1:2]) block_info_ptr
+	// raw_dev is used to allow for multiple bindings per process.
+	struct stat sb;
+	int result = fstat(fd, &sb);
+	if (result == -1) {
+		return -1;
+	}
+	struct raw_config_request* rq = (struct raw_config_request*)malloc(sizeof(struct raw_config_request));
+	if (rq == NULL) {
+		return -1;
+	}
+	// +1 for /dev/raw/rawN, as N must be greater than 0
+	rq->raw_minor = procid + 1;
+#if SYZ_MULTI_PROC
+	rq->raw_minor += /*PROCS*/ *arg;
+#endif
+	rq->block_major = major(sb.st_rdev);
+	rq->block_minor = minor(sb.st_rdev);
+	debug("syz_raw_bind_info rq{%d (%d:%d)} fd: %d\n", (int)rq->raw_minor, (int)rq->block_major, (int)rq->block_minor, (int)fd);
+	return (long)rq;
+}
+#endif
+
+#if SYZ_EXECUTOR || __NR_syz_prepare_loop_device
+#include <fcntl.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+static long syz_prepare_loop_device(int loopNo)
+{
+	int file_id = procid;
+#if SYZ_MULTI_PROC
+	file_id += /*PROCS*/ *loopNo;
+#endif
+
+	char tmpFileName[64];
+	snprintf(tmpFileName, sizeof(tmpFileName), "/tmp/fileRAW%d", file_id);
```

Comment:
We can't use files at global locations. Nobody will cleanup after this, this will make programs collide and non-reproducible and /tmp is not necessary even mounted everywhere.